docker/dev/docker-compose.yml

```diff
@@ -46,13 +46,13 @@ services:
       - pictrs
 
   lemmy-ui:
-    image: dessalines/lemmy-ui:dev
+    # image: dessalines/lemmy-ui:dev
     # use this to build your local lemmy ui image for development
     # run docker compose up --build
     # assuming lemmy-ui is cloned besides lemmy directory
-    # build: 
-    #  context: ../../../lemmy-ui
-    #  dockerfile: Dockerfile  
+    build: 
+     context: ../../../lemmy-ui
+     dockerfile: Dockerfile  
```

Comment:
This will break if someone doesnt have lemmy-ui cloned, or if its cloned in a different location. Maybe do the docker build conditionally in bash script, only if the folder exists.

Reply:
I can just change this one back, I agree its only necessary if you're doing front end dev.